specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2020-09-01/storagesync.json

```diff
@@ -28,7 +28,7 @@
         "operationId": "Operations_List",
         "parameters": [
           {
-            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
```

Comment:
Except ErrorResponse,  all other common definition like ApiVersionParameter SubscriptionIdParameter can still ref to v1.

Only change in V2 is ErrorResponse.

Reply:
Ok, set references back to v1